README.md

```diff
@@ -31,6 +31,41 @@ To sort or remove unused imports you can trigger the `LSP-typescript: Organize I
 
 The `LSP-typescript: Find Callers` command can be used to find what is calling the given symbol. It has some overlap with the built-in `LSP: Find References` command but returns only the places where the symbol was called.
 
+
+## Inlay hints (experimental)
+
+Inlay hints are short textual annotations that show parameter names, type hints.
+
+![inlay-hints](/images/inlay-hints.png)
+
+Inlay hints are disabled by default.
+To enable them, modify the following settings in `LSP-typescript.sublime-settings`:
```

Comment:
The configuration file name probably is not very telling for less advanced users. Could instead say to do it through `Preferences: LSP-typescript Settings`